reqs/requirements-dev-lite.txt

```diff
@@ -8,7 +8,7 @@ nbconvert>=5
 nbformat>=5.0
 pact-python>=2.0.1
 pyfakefs>=4.5.1
-pytest>=6.2.0
+pytest>=7.2,<8.0
```

Comment:
Could we add a comment around why we're pinning?

Reply:
Added